reVX/least_cost_xmission/least_cost_paths.py

```diff
@@ -380,3 +384,248 @@ def run(cls, cost_fpath, features_fpath, capacity_class,
                             (time.time() - ts) / 3600))
 
         return least_cost_paths
+
+
+class ReinforcementPaths(LeastCostPaths):
+    """
+    Compute reinforcement line paths between substations and a single
+    balancing area network node.
+    """
+    def __init__(self, cost_fpath, features, transmission_lines,
+                 xmission_config=None):
+        """
+        Parameters
+        ----------
+        cost_fpath : str
+            Path to h5 file with cost rasters and other required layers.
+        features_fpath : str
```

Comment:
Can you clarify this docstring? Is this supposed to be a filepath or a geodataframe? I think it get init from run() with a gdf?

Reply:
Great catch! Updated below